README.md

```diff
@@ -164,99 +188,71 @@ Read more about the semantic versioning [here](https://docs.npmjs.com/getting-st
+#### 1. Create a branch for the new release
 
-Release PRs **MUST** be named with `release-` prefix. This will kick off the AWS integration tests in the CI process and ensures that package updates are fully tested on AWS before publication to NPM.
+The name is not important, but `release-x.y.z` seems like a good choice.
 
-#### 2. Update the Cumulus Version number
+#### 2. Update the Cumulus version number
 
-When changes are ready to be released, the Cumulus version number must be updated using semantic versioning.
+When changes are ready to be released, the Cumulus version number must be
+updated.
 
-Lerna handles the process of deciding which version number should be used as long as the developer decides whether the change is a patch or a minor/major change.
+Lerna handles the process of deciding which version number should be used as
+long as the developer specifies whether the change is a major, minor, or patch
+change.
 
 To update cumulus' version number run:
 
      $ yarn update
 
-You will be prompted to select the type of change (patch/minor/major). Lerna will update the version of all packages after the selection.
-
 ![](https://static.notion-static.com/13acbe0a-c59d-4c42-90eb-23d4ec65c9db/Screen_Shot_2018-03-15_at_12.21.16_PM.png)
 
-#### 3. Update the Changelog
+#### 3. Update CHANGELOG.md
 
-Update the CHANGELOG.md. Put a header under the 'Unreleased' section with the new version number and the date.
+Update the CHANGELOG.md. Put a header under the 'Unreleased' section with the
+new version number and the date.
 
-Add a link reference for the github "compare" view at the bottom of the CHANGELOG.md, following the existing pattern. This link reference should create a link in the CHANGELOG's release header to changes in the corresponding release.
+Add a link reference for the github "compare" view at the bottom of the
+CHANGELOG.md, following the existing pattern. This link reference should create
+a link in the CHANGELOG's release header to changes in the corresponding
+release.
 
-#### 4. Update the example package.json
+#### 4. Update example/package.json
 
 Update example/package.json to point to the new Cumulus packages.
 
-#### 5. Create a git tag
-
-The CHANGELOG changes and package updates should be pushed to git. Then tag the release. The tag is what tells npm what to publish, so it is important for this branch to be up to date with master and have all these changes pushed.
-
-Create a new git tag
-
-  $ git tag -a v1.x.x -m "version 1.x.x release"
+#### 5. Create a pull request against the master branch
 
-Push the tag to github
+Create a PR against the `master` branch. Verify that the Travis CI build for the
+PR succeeds.  Once merged, the release branch can be deleted.
 
-  $ git push origin v1.x.x
+#### 6. Create a git tag for the release
 
-#### 6. PR and merge to master
+Publishing of new releases is handled by Travis CI and is triggered by the
```

Comment:
Where in that process should this be done? Right after merge to master? I think we should specify that pushing the tag actually triggers the release build.

Reply:
Yes, it should be done after the merge to master.  I've updated steps 5 and 6 to make it more explicit.